readthedocs/search/faceted_search.py

```diff
@@ -216,9 +209,21 @@ def _is_advanced_query(self, query):
         query_tokens = set(query)
         return not tokens.isdisjoint(query_tokens)
 
+
+class ProjectSearch(RTDFacetedSearch):
+    facets = {'language': TermsFacet(field='language')}
+    doc_types = [ProjectDocument]
+    index = ProjectDocument._index._name
+    fields = ('name^10', 'slug^5', 'description')
+    operators = ['and', 'or']
+    excludes = ['users', 'language']
+
     def query(self, search, query):
```

Comment:
This query method was moved from the parent to this class, as it has specifics about the contents of this class, and it's overridden by the other class that inherits it.